docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -399,20 +399,54 @@ httpCall()
 
 .. code-block:: lua
 
-  local headers, body = handle:httpCall(cluster, headers, body, timeout, asynchronous)
+  local headers, body = handle:httpCall(cluster, headers, body, timeout, async)
+
+  -- Alternative function signature.
+  local headers, body = handle:httpCall(cluster, headers, body, options)
 
 Makes an HTTP call to an upstream host. *cluster* is a string which maps to a configured cluster manager cluster. *headers*
 is a table of key/value pairs to send (the value can be a string or table of strings). Note that
 the *:method*, *:path*, and *:authority* headers must be set. *body* is an optional string of body
 data to send. *timeout* is an integer that specifies the call timeout in milliseconds.
 
-*asynchronous* is a boolean flag. If asynchronous is set to true, Envoy will make the HTTP request and continue,
+*async* is a boolean flag. If async is set to true, Envoy will make the HTTP request and continue,
 regardless of the response success or failure. If this is set to false, or not set, Envoy will suspend executing the script
 until the call completes or has an error.
 
 Returns *headers* which is a table of response headers. Returns *body* which is the string response
 body. May be nil if there is no body.
 
+
+The alternative function signature allows caller to specify *options* as a table. Currently,
+the supported keys are:
+
+- *async* is a boolean flag that controls the asynchronicity of the HTTP call.
+  It refers to the same *async* flag as the first function signature.
+- *timeout* is an integer that specifies the call timeout in milliseconds.
+  It refers to the same *timeout* argument as the first function signature.
+- *sampled* is a boolean flag that decides whether the produced trace span will be sampled or not.
+- *multiple* is boolean flag that decides whether the returned headers could have multiple values for same header name.
```

Comment:
multiple_values_per_response_header? (Or something else that is a bit more clear what this is?)

Can you also clarify the format of the returned headers both when this is set to false and true? (From reading I'm not sure what happens here in the false case, and in the true case is it comma delimited or something else?)

For both of the names changes please remember to update the release notes also.

Reply:
I will add some more comments. `may_multiple_values_per_response_header` would be better, but seem it's too long. 🤣